src/test-helpers.tsx

```diff
@@ -174,25 +175,36 @@ export const createVersionWithEntries = (
   return createInternalVersion(createExternalVersionWithEntries(...args));
 };
 
+export const createVersionAndEntryStatusMap = (
+  ...args: Parameters<typeof createExternalVersionWithEntries>
+) => {
+  return {
+    version: createVersionWithEntries(...args),
```

Comment:
This relies on the implementation of `createVersionWithEntries` a little too much. The two objects might get out of sync. How about just explicitly using the same object?

```diff
diff --git a/src/test-helpers.tsx b/src/test-helpers.tsx
index 82765da..45a61b1 100644
--- a/src/test-helpers.tsx
+++ b/src/test-helpers.tsx
@@ -178,11 +178,10 @@ export const createVersionWithEntries = (
 export const createVersionAndEntryStatusMap = (
   ...args: Parameters<typeof createExternalVersionWithEntries>
 ) => {
+  const externalVersion = createExternalVersionWithEntries(...args);
   return {
-    version: createVersionWithEntries(...args),
-    entryStatusMap: createEntryStatusMap(
-      createExternalVersionWithEntries(...args),
-    ),
+    version: createInternalVersion(externalVersion),
+    entryStatusMap: createEntryStatusMap(externalVersion),
   };
 };
 
```